modules/programs/eza.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta.maintainers = [ maintainers.cafkafk ];
```

Comment:
I don't think `maintainers` is in scope here (also doesn't seem like you added yourself to `modules/lib/maintainers.nix`

Reply:
> I don't think maintainers is in scope here

Sorry, I think I was too fast, changed it to `lib.pkgs.maintainers`. 

> also doesn't seem like you added yourself to modules/lib/maintainers.nix

The docs said you didn't have to add yourself if you were in the nixpkgs maintainers file. If I can't use nixpkgs I'll add myself to the home manager file.